curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff732ad5c56c1a497c7f16.md

```diff
@@ -0,0 +1,138 @@
+---
+id: 63ff732ad5c56c1a497c7f16
+title: Step 5
+challengeType: 0
+dashedName: step-5
+---
+
+# --description--
+
+Apart from the `.then()` method, the `.catch()` method is another asynchronous JavaScript method with which you can handle errors. This is useful in case the `Promise` gets rejected.
+
+Add a `.catch()` to the last `.then()`. Set the error variable to `err` inside the `catch()`, add your callback, then log that error to the console, for now.
```

Comment:
In the next step, it looks like the code using `console.error` with a custom message

```
 .catch((err) => {
    console.error(`There was an error: ${err}`);
  });
```
I think we should either update the code or directions for consistency.

Reply:
I don't understand what you mean here. Are you saying the JS seed content of this step and the next one are the same?

Reply:
If you look at the code in the next step, it shows the use of a `console.error` with the message of `There was an error: ${err}`  but your directions here don't mention that.

Reply:
Oh. Thanks for the clarification.